ajax/libs/cellx/package.json

```diff
@@ -18,7 +18,7 @@
     {
       "basePath": "dist",
       "files": [
-        "**/*"
+        "cellx.min.js"
```

Comment:
So the new auto-update config will only fetch one file? But looks like you added multi assets?